src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -63,96 +65,103 @@ public boolean doesUserExist(final String userName) {
       ps.close();
       return found;
     } catch (final SQLException sqle) {
-      s_logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
+      logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
       throw new IllegalStateException(sqle.getMessage());
     } finally {
       DbUtil.closeConnection(con);
     }
   }
 
+  private static String mask(final HashedPassword hashedPassword) {
+    return mask(hashedPassword.value);
+  }
+
+  private static String mask(final String hashedPassword) {
+    return hashedPassword.replaceAll(".", "*");
```

Comment:
Thoughts on moving `mask` to `HashedPassword`? This way it is not a random utility function, and testing it makes more sense.